packages/atomic/src/components/common/atomic-modal/atomic-modal.tsx

```diff
@@ -109,13 +110,28 @@ export class AtomicModal implements InitializableComponent<AnyBindings> {
     return classes;
   }
 
+  private handleCloseOnEscape() {
+    this.closeOnEscape = (e: KeyboardEvent) => {
+      if (e.key.toLowerCase() === 'escape') {
+        this.close();
+      }
+    };
+    document.body.addEventListener('keyup', this.closeOnEscape);
```

Comment:
Can this be replaced with a `@Listen` decorator with the `document` target? https://stenciljs.com/docs/events#target

Reply:
🤔 Yeah probably !